### PR TITLE
Verify commit signoff GHA

### PR DESCRIPTION
**Pull request by AnneCYH**
Add GitHub Action workflow that checks that commits are signed off (-s or --signoff).

### CODE REVIEW

**Review by benjamin051000 — CHANGES_REQUESTED** [review]

File: .github/workflows/verify-commit-signoff.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Verify Commit Signoff
+
+on:
+  push:
```
[the remark this answers is not in the document]
Comment (AnneCYH) [reply .github/workflows/verify-commit-signoff.yml:4]: We can change that for sure. Less resource intensive this way also.

[the remark this answers is not in the document]
Comment (AnneCYH) [reply .github/workflows/verify-commit-signoff.yml]: hey @benjamin051000 where is this action defined?
https://kernel.ubuntu.com/gitea/kernel/kteam-tools/src/commit/6b7f4e2c20902939d1c31461ed414290240862fe/.gitea/workflows/pr_check.yaml#L89

the verify-signoff workflow here doesn't seem to be robust enough when i squash commits; it's grabbing the wrong commit ID to check

[the remark this answers is not in the document]
Comment (AnneCYH) [reply .github/workflows/verify-commit-signoff.yml:4]: it should be working as expected now.
tested with a few squashes and pushes.

Thanks.

[the remark this answers is not in the document]
Comment (benjamin051000) [reply .github/workflows/verify-commit-signoff.yml:4]: Hey @AnneCYH, I've been on PTO for the past 5 days, lol. The action is defined here: https://kernel.ubuntu.com/gitea/actions/tnt-sign-off-check. I'm not sure if you found that before.

[the remark this answers is not in the document]
Comment (benjamin051000) [reply .github/workflows/verify-commit-signoff.yml:4]: This looks better to me. Thanks for that update.

**Review by benjamin051000 — APPROVED** [review]
Looks like this is working as intended, LGTM.

**Pull request merged**